### ISSUE TITLE
Writing Windows 10 image for UEFI mode...

### ISSUE THREAD

**Issue by Jakecheslekov** [MartinBriza/MediaWriter#156]
Hey Martin, don't know if you develop this for all ISO's or just Linux, its fine if you don't support other ISO's, but I noticed that Windows 10 doesn't burn correctly for UEFI, as UEFI mode on two different PCs doesn't seem to recognize the USB once created, but Linux UEFI works out just fine with your program. Also seems NetBSD & Solaris don't seem to burn to USB properly either.

**Comment by MartinBriza** [comment]
Yeah, I know it doesn't write Windows properly. I'm specifically targetting Fedora with Fedora Media Writer. It copies the images to the drive bit-by-bit, in the same way dd does it (but with slight improvements to the process, like being able to do that without root access and being able to check the written data). This works for some other distros and OSes (like FreeDOS for example, I'm told), however for some others, like NetBSD, as you've said, it does not.
The main point is though, I'm not planning to support any other OS or distro in the near future.
